Controller/AbstractObjectTranslationController.php

```diff
@@ -26,21 +30,25 @@ public static function translate($translationObject)
             return $retArray;
         }
 
-        $translationProblemArray = [];
-
         foreach ($translationObject->translationModel as $key => $value) {
-            //check against values
-            if (!self::checkOverwrite($translationObject, $value)) { //retain the value if set not to overwrite
-                self::setValue($translationObject->toObject, $translationObject->toObject, $value, $value); // TODO: ($value, $value) is this the correct argument here? Method signature defines $key, $value
```

Comment:
In cases of failure, setValue returned an array, otherwise it returned nothing. The issue was that there was no way to retrieve the actual values from the array. Added try, catch block, and changed the error collection to a static property for the controller.